<ui version="4.0" >
 <class>imageDialog</class>
 <widget class="QDialog" name="imageDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Insert Image....</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Image</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Image URL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Image Descrption</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Alignment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Dimensions</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLineEdit" name="txtURL" />
          </item>
          <item>
           <widget class="QLineEdit" name="txtDesc" />
          </item>
          <item>
           <widget class="QComboBox" name="comboAlignment" >
            <item>
             <property name="text" >
              <string>-- Not Set --</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Baseline</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Top</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Middle</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Bottom</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Text Top</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Absolute Middle</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Absolute Bottom</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Left</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Right</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QSpinBox" name="spinX" />
            </item>
            <item>
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinY" />
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QPushButton" name="insertBttn" >
          <property name="text" >
           <string>Insert</string>
          </property>
          <property name="icon" >
           <iconset>/usr/share/pixmaps/chotha/ok.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>21</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBttn" >
          <property name="text" >
           <string>Cancel</string>
          </property>
          <property name="icon" >
           <iconset>/usr/share/pixmaps/chotha/dialog-cancel.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
